examples/hil.cfg.dev-no-hardware

```diff
@@ -26,18 +26,7 @@ dry_run=True
 [extensions]
 hil.ext.switches.mock =
 hil.ext.obm.mock =
-
 hil.ext.network_allocators.null =
-
-# Depending on what you're trying to do, you may want to use the vlan_pool
-# network allocator instead of the null allocator. To do this, comment out the
-# null allocator extension above, and uncomment the following:
-#
-# hil.ext.network_allocators.vlan_pool =
-#
-# [hil.ext.network_allocators.vlan_pool]
-# vlans = 100-200, 300-500
-
 hil.ext.auth.null =
 # Again, depending on what you're doing, you may want to change this to a
```

Comment:
Minor nit: You can remove "again" from the starting of this line now, since this paragraph precedes the vlan_pool one which you moved.

Reply:
Fix this and I'm happy.